core/constants_test.py

```diff
@@ -64,6 +65,26 @@ def test_loading_file_in_package_returns_the_content(self) -> None:
                 'File data'
             )
 
+        with self.swap_to_always_return(pkgutil, 'get_data', 'File data'):
+            self.assertEqual(
+                constants.get_package_file_contents(
+                    'assets', 'non_exist.xy', binary_mode=True), 'File data'
+            )
+
+        with utils.open_file(
+            os.path.join(
+                'assets', 'images', 'avatar', 'user_blue_150px.png'),
+            'rb',
+            encoding=None
+        ) as f:
+            raw_image_png = f.read()
+        default_image_path = os.path.join(
+            'images', 'avatar', 'user_blue_150px.png')
+        self.assertEqual(
+            constants.get_package_file_contents(
+                'assets', default_image_path, binary_mode=True), raw_image_png
+        )
+
```

Comment:
Please add separate test functions and do not extend the existing ones.

Reply:
Done